subiquity/server/types.py

```diff
@@ -22,3 +22,4 @@ class InstallerChannels(CoreChannels):
     GEOIP = 'geoip'
     CONFIGURED = 'configured'
     APT_CONFIGURED = 'apt-configured'
+    SOURCE_VARIANT_SET = 'client-variant-set'
```

Comment:
nitpick - one says source, one says client

Reply:
nice catch, thanks! I obviously couldn't decide which one to use :D